src/commonMain/kotlin/com/algolia/search/endpoint/EndpointSearch.kt

```diff
@@ -122,25 +122,37 @@ public interface EndpointSearch {
         requestOptions: RequestOptions? = null
     ): ResponseSearch
 
+    @Deprecated(
+        message = "Use findObject instead.",
+        replaceWith = ReplaceWith("findObject(match, query, doNotPaginate, requestOptions)"),
+        level = DeprecationLevel.ERROR
```

Comment:
It's not an warning?

Reply:
`ERROR` will force developers to change it asap, as it prevents compilation. 

Maybe it's too harsh, I will lower it to `WARNING`